mcs/class/System.Drawing/System.Drawing.Configuration/SystemDrawingSection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
```

Comment:
You need to add this file to System.Drawing.dll.sources so it is compiled (and also add a reference to System.Configuration to LIB_REFS).